scripts/servers_test.py

```diff
@@ -540,20 +567,22 @@ def mock_os_path_exists(path):
 
         self.exit_stack.close()
 
-    def test_managed_redis_server_deletes_redis_dump_when_it_exists(self):
+    def test_managed_redis_server_deletes_redis_dump_when_it_exists(
+        self
+    ) -> None:
         original_os_remove = os.remove
         original_os_path_exists = os.path.exists
 
         @test_utils.CallCounter
-        def mock_os_remove(path):
+        def mock_os_remove(path: str) -> None:
             if path == common.REDIS_DUMP_PATH:
                 return
             original_os_remove(path)
 
-        def mock_os_path_exists(path):
+        def mock_os_path_exists(path: str) -> bool:
             if path == common.REDIS_DUMP_PATH:
                 return True
-            original_os_path_exists(path)
+            return original_os_path_exists(path)
```

Comment:
This changes the behavior of the mock. Why?

Reply:
Fixed by returning None at the end.